database/query/provider_github_app_installations.sql

```diff
@@ -1,6 +1,9 @@
 -- name: GetInstallationIDByProviderID :one
 SELECT * FROM provider_github_app_installations WHERE provider_id = $1;
 
+-- name: GetInstallationIDByAppID :one
```

Comment:
non-blocking: A better name for this would be `GetInstallationByInstallationID`, since app ID is something else.

Reply:
thank you I will take this into account